src/components/FeedbackButton/FeedbackButton.js

```diff
@@ -4,29 +4,41 @@ import classNames from 'classnames';
 import { NavLink } from 'react-router-dom';
 import { LINKS } from '../../constants';
 import Icon from '../Icon/Icon';
+import {
+  hasVisitedFeedbackPage,
+  navigationCount,
+} from '../../helpers/userBehaviour';
 import style from './FeedbackButton.scss';
 
+const PULSE_ON_NUM_NAVIGATIONS = 3;
+
 const FeedbackButton = (props) => {
+  // If user has not yet visited the feedback page, pulse every few navigations
+  const shouldPulse = !hasVisitedFeedbackPage()
+    && navigationCount() > 0
+    && navigationCount() % PULSE_ON_NUM_NAVIGATIONS === 0;
+
   const buttonClassName = classNames(
     style.button,
     {
       [style.dimmed]: props.dimmed,
+      [style.pulsing]: shouldPulse,
     },
   );
 
   return (
-    <div className={style.buttonWrapper}>
-      <NavLink to={LINKS.FEEDBACK}>
-        <div className={buttonClassName}>
-          <Icon
-            icon="speechBubble"
-            className={style.speechBubbleIcon}
-            size={25}
-          />
-          Feedback
-        </div>
-      </NavLink>
-    </div>
+    <NavLink to={LINKS.FEEDBACK}>
+      <div className={buttonClassName}>
+        <span className={style.buttonText}>
+          Give feedback
+        </span>
+        <Icon
+          icon="speechBubble"
+          className={style.speechBubbleIcon}
+          size={25}
+        />
+      </div>
+    </NavLink>
```

Comment:
The `<NavLink>` component will apply a `className` that you pass it, so you don't need the wrapper `<div>`.
```jsx
<NavLink
  to={LINKS.FEEDBACK}
  className={buttonClassName}
>
  <span className={style.buttonText}>
    Give feedback
  </span>

  <Icon
    icon="speechBubble"
    className={style.speechBubbleIcon}
    size={25}
  />
</NavLink>
```

Pretty sure that'll work.